Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -300,8 +301,31 @@ its :php:`__invoke()` method will be called:
 
 Read :ref:`how to configure dependency injection in extensions <dependency-injection-in-extensions>`.
 
+Existing event listeners can be overridden by custom implementations. For example,
+:file:`EXT:news/Configuration/Services.yaml` could listen on the event
+:php:`TYPO3\CMS\Frontend\Event\ModifyHrefLangTagsEvent`:
 
+..  literalinclude:: _ServicesOverrideBase.yaml
+    :language: yaml
+    :caption: EXT:my_extension/Configuration/Services.yaml
+
+If you want to replace this event listener with your custom implementation, your extension can
+achieve this by specifying:
+
+..  literalinclude:: _ServicesOverrideBase.yaml
```

Comment:
Two times `_ServicesOverrideBase.yaml` - should this not be the `_ServicesOverrideCustom.yaml` for the second one ?

Reply:
Yes, copypasta, thanks!